Template/azure/sqlserver.json

```diff
@@ -45,7 +45,22 @@
             "properties": {
                 "administratorLogin": "[parameters('adminUsername')]",
                 "administratorLoginPassword": "[parameters('adminPassword')]"
-            }
+            },
+            "resources": [
+                {
+                    "type": "firewallRules",
+                    "apiVersion": "2014-04-01",
+                    "dependsOn": [
+                        "[resourceId('Microsoft.Sql/servers', parameters('sqlServername'))]"
+                    ],
+                    "location": "[resourceGroup().location]",
+                    "name": "AllowAllWindowsAzureIps",
+                    "properties": {
+                        "startIpAddress": "0.0.0.0",
+                        "endIpAddress": "0.0.0.0"
+                    }
+                }
+            ]
```

Comment:
the sql server requires the above firewall rule for Import/Export to connect to azure resources and services like azure storage account

Reply:
I do not like it being open to all Azure addresses. Why can it not be restricted to the ip address that is executing the commands?